### ISSUE TITLE
Does not work with the newest gradle version it seems

### ISSUE THREAD

**Issue by derekpitts** [ByneappLLC/flutter_config#55]
Tried to run with the most recent version of gradle plugin:

Running "flutter pub get" in adventure_pal...
Launching lib\app\main.dart on Pixel 6 in debug mode...
Running Gradle task 'assembleDebug'...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':flutter_config'.
> Could not resolve all files for configuration ':flutter_config:classpath'.
   > Could not find org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.7.10/kotlin-gradle-plugin-1.7.10.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config
   > Could not find org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.71.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.71/kotlin-stdlib-jdk8-1.3.71.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.lint:lint-gradle-api:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:gradle-api:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:common:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:manifest-merger:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.analytics-library:tracker:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools:repository:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4 > com.android.tools.analytics-library:shared:26.6.4
   > Could not find org.ow2.asm:asm:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm/7.0/asm-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find org.ow2.asm:asm-analysis:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-analysis/7.0/asm-analysis-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
   > Could not find org.ow2.asm:asm-commons:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-commons/7.0/asm-commons-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find org.ow2.asm:asm-util:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find net.sf.jopt-simple:jopt-simple:4.9.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/net/sf/jopt-simple/jopt-simple/4.9/jopt-simple-4.9.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find net.sf.proguard:proguard-gradle:6.0.3.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/net/sf/proguard/proguard-gradle/6.0.3/proguard-gradle-6.0.3.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
   > Could not find com.google.protobuf:protobuf-java:3.4.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/protobuf/protobuf-java/3.4.0/protobuf-java-3.4.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:bundletool:0.10.3
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.analytics-library:protos:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.analytics-library:tracker:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:bundletool:0.10.3 > com.android.tools.build:aapt2-proto:0.4.0
   > Could not find com.google.protobuf:protobuf-java-util:3.4.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/protobuf/protobuf-java-util/3.4.0/protobuf-java-util-3.4.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:bundletool:0.10.3
   > Could not find com.squareup:javawriter:2.5.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/squareup/javawriter/2.5.0/javawriter-2.5.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find org.bouncycastle:bcpkix-jdk15on:1.56.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/bouncycastle/bcpkix-jdk15on/1.56/bcpkix-jdk15on-1.56.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:apkzlib:3.6.4
   > Could not find org.bouncycastle:bcprov-jdk15on:1.56.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/bouncycastle/bcprov-jdk15on/1.56/bcprov-jdk15on-1.56.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:apkzlib:3.6.4
   > Could not find org.ow2.asm:asm-tree:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-tree/7.0/asm-tree-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find javax.inject:javax.inject:1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/javax/inject/javax.inject/1/javax.inject-1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
   > Could not find it.unimi.dsi:fastutil:7.2.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/it/unimi/dsi/fastutil/7.2.0/fastutil-7.2.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find com.googlecode.json-simple:json-simple:1.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/googlecode/json-simple/json-simple/1.1/json-simple-1.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4
   > Could not find com.google.guava:guava:27.1-jre.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/guava/guava/27.1-jre/guava-27.1-jre.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.analytics-library:crash:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.lint:lint-gradle-api:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:gradle-api:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:common:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.analytics-library:tracker:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4 > com.android.tools.analytics-library:shared:26.6.4
   > Could not find org.apache.httpcomponents:httpmime:4.5.6.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/apache/httpcomponents/httpmime/4.5.6/httpmime-4.5.6.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.analytics-library:crash:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4
   > Could not find org.apache.httpcomponents:httpcore:4.4.10.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/apache/httpcomponents/httpcore/4.4.10/httpcore-4.4.10.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.analytics-library:crash:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4
   > Could not find org.apache.httpcomponents:httpclient:4.5.6.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/apache/httpcomponents/httpclient/4.5.6/httpclient-4.5.6.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.analytics-library:crash:26.6.4
   > Could not find org.jetbrains.kotlin:kotlin-reflect:1.3.71.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-reflect/1.3.71/kotlin-reflect-1.3.71.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.lint:lint-gradle-api:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
   > Could not find org.antlr:antlr4:4.5.3.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/antlr/antlr4/4.5.3/antlr4-4.5.3.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
   > Could not find commons-io:commons-io:2.4.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/commons-io/commons-io/2.4/commons-io-2.4.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
   > Could not find com.googlecode.juniversalchardet:juniversalchardet:1.0.3.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/googlecode/juniversalchardet/juniversalchardet/1.0.3/juniversalchardet-1.0.3.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
   > Could not find com.squareup:javapoet:1.10.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/squareup/javapoet/1.10.0/javapoet-1.10.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
   > Could not find com.google.code.gson:gson:2.8.5.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/code/gson/gson/2.8.5/gson-2.8.5.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > androidx.databinding:databinding-compiler-common:3.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:manifest-merger:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4 > com.android.tools.analytics-library:shared:26.6.4
   > Could not find com.google.auto.value:auto-value-annotations:1.6.2.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/auto/value/auto-value-annotations/1.6.2/auto-value-annotations-1.6.2.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:bundletool:0.10.3
   > Could not find com.google.errorprone:error_prone_annotations:2.3.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/errorprone/error_prone_annotations/2.3.1/error_prone_annotations-2.3.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:bundletool:0.10.3
   > Could not find com.google.guava:guava:27.1-jre.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/guava/guava/27.1-jre/guava-27.1-jre.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:bundletool:0.10.3
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android:zipflinger:3.6.4
   > Could not find com.google.code.gson:gson:2.8.5.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/code/gson/gson/2.8.5/gson-2.8.5.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta08
   > Could not find org.jetbrains.kotlin:kotlin-stdlib:1.3.41.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib/1.3.41/kotlin-stdlib-1.3.41.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta08
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta08
   > Could not find org.ow2.asm:asm:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm/7.0/asm-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta08
   > Could not find org.ow2.asm:asm-util:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta08
   > Could not find org.ow2.asm:asm-commons:7.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-commons/7.0/asm-commons-7.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta08
   > Could not find org.jdom:jdom2:2.0.6.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jdom/jdom2/2.0.6/jdom2-2.0.6.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta08
   > Could not find org.apache.commons:commons-compress:1.12.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/apache/commons/commons-compress/1.12/commons-compress-1.12.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools:repository:26.6.4
   > Could not find org.jetbrains.trove4j:trove4j:20160824.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdk-common:26.6.4
   > Could not find net.sf.kxml:kxml2:2.3.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/net/sf/kxml/kxml2/2.3.0/kxml2-2.3.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:manifest-merger:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.ddms:ddmlib:26.6.4
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools.layoutlib:layoutlib-api:26.6.4
   > Could not find com.google.code.findbugs:jsr305:1.3.9.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/code/findbugs/jsr305/1.3.9/jsr305-1.3.9.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:apkzlib:3.6.4
   > Could not find com.google.guava:guava:27.1-jre.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/guava/guava/27.1-jre/guava-27.1-jre.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools.build:apkzlib:3.6.4
   > Could not find org.jetbrains:annotations:13.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/annotations/13.0/annotations-13.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools.layoutlib:layoutlib-api:26.6.4
   > Could not find com.sun.activation:javax.activation:1.2.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/sun/activation/javax.activation/1.2.0/javax.activation-1.2.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools:repository:26.6.4
   > Could not find org.glassfish.jaxb:jaxb-runtime:2.3.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/glassfish/jaxb/jaxb-runtime/2.3.1/jaxb-runtime-2.3.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools:repository:26.6.4
   > Could not find com.google.jimfs:jimfs:1.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/jimfs/jimfs/1.1/jimfs-1.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :flutter_config > com.android.tools.build:gradle:3.6.4 > com.android.tools.build:builder:3.6.4 > com.android.tools:sdklib:26.6.4 > com.android.tools:repository:26.6.4
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':flutter_config' of type org.gradle.api.Project.
   > Could not get unknown property 'android' for project ':flutter_config' of type org.gradle.api.Project.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by CharanjeetRPOPS** [comment]
Facing same issue

**Comment by stevennguyenn** [comment]
https://pub.dev/packages/flutter_config_plus
I just publish a copy library. You can use it 👯

**Comment by derekpitts28** [comment]
Checking in to see if we can get the kotlin version bumped up to latest.

**Comment by elhe26** [comment]
@stevennguyenn your library is crashing on iOS.

specifically, this line:

```swift
    var mapKey = FlutterConfigPlusPlugin.env(for: "GOOGLE_MAPS_API_KEY")
```

**Comment by edTheGuy00** [comment]
Fixed with #57
